trimmer_decorator.rb

```diff
@@ -0,0 +1,5 @@
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.length > 9 ? "#{@nameable.correct_name[0..8]} ..." : @nameable.correct_name
```

Comment:
- [ ] This Trimmer returns only 9 characters where you are expected to return 10 characters. Kindly do that.

  You can change `[0..8]` to `[0..9]` to resolve this.

- [ ] Note that you simply expected to return the trimmed correct_name, not a string particularly. Kindly remove the string in the conditional

Reply:
Thank you for your feedback, I just fixed the mistakes.